packages/components/src/navigation/stories/controlled-state.js

```diff
@@ -0,0 +1,123 @@
+/**
+ * WordPress dependencies
+ */
+import { Button } from '@wordpress/components';
+import { useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import Navigation from '..';
+import NavigationItem from '../item';
+import NavigationMenu from '../menu';
+
+export function ControlledStateStory() {
+	const [ activeItem, setActiveItem ] = useState( 'item-1' );
+	const [ activeMenu, setActiveMenu ] = useState( 'root' );
+
+	// Mock navigation link
+	const Link = ( { href, children } ) => (
```

Comment:
Wondering if we should rename this to `MockLink` or `FakeLink`? 🤔

Reply:
They have pretty much the same meaning 🤷 😄

Reply:
Yeah, I mean, `Link` sounds like a real component 😄

Reply:
Oooh, I totally misread your suggestion! 😄 
Yeah, I agree, it would make it definitely clear that's a _mock link_.